mongodb-panache-quickstart/src/test/java/org/acme/mongodb/panache/PersonResourceTest.java

```diff
@@ -0,0 +1,187 @@
+package org.acme.mongodb.panache;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.config.ObjectMapperConfig;
+import io.restassured.http.ContentType;
+import io.restassured.parsing.Parser;
+import org.assertj.core.api.Assertions;
+import org.bson.types.ObjectId;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.time.LocalDate;
+
+import static io.restassured.config.LogConfig.logConfig;
+import static org.acme.mongodb.panache.MongoDbContainer.MONGODB_HOST;
+import static org.acme.mongodb.panache.MongoDbContainer.MONGODB_PORT;
+
+@Testcontainers
+@QuarkusTest
+class PersonResourceTest {
+
+    @Container
+    static GenericContainer MONGO_DB_CONTAINER = new MongoDbContainer()
+            .withCreateContainerCmdModifier(cmd -> {
+                cmd.withHostName(MONGODB_HOST)
+                        .withPortBindings(new PortBinding(Ports.Binding.bindPort(MONGODB_PORT), new ExposedPort(MONGODB_PORT)));
+            });
+
+    @BeforeAll
+    static void initAll() {
+        RestAssured.defaultParser = Parser.JSON;
+        RestAssured.config
+                .logConfig((logConfig().enableLoggingOfRequestAndResponseIfValidationFails()))
+                .objectMapperConfig(new ObjectMapperConfig().jackson2ObjectMapperFactory((type, s) -> new ObjectMapper()
+                        .registerModule(new Jdk8Module())
+                        .registerModule(new JavaTimeModule())
+                        .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)));
+    }
+
+    @Test
+    void test() {
+        PersonDTO person1 = PersonDTO.Builder
+                .aPersonDTO()
+                .withId("5889273c093d1c3e614bf2f9")
```

Comment:
ID will be automatically populated by MongoDB so you can skip them

Reply:
The rest service should return them inside the location header so you can retrieve them from restassured response.

Reply:
I my tests, there is nothing returned in headers by keeping the actual behaviour.
![headers](https://user-images.githubusercontent.com/22281426/82729984-aad50400-9cfc-11ea-9fe9-c08d48e99b87.png)